include/mxnet/mem_mgr.h

```diff
@@ -0,0 +1,17 @@
+#include <cuda_runtime_api.h>
+#include <stdio.h>
+#include "storage.h"
```

Comment:
I think `include storage.h` can be removed if it is not used. Same applies to other headers